application/models/Question.php

```diff
@@ -266,6 +266,16 @@ public function getAdvancedSettingsWithValues($iQuestionID, $sQuestionType, $iSu
             return $aAttributeNames;
         }
 
+        function getQuestionById($qid)
```

Comment:
There is no need for this function.
1. It does not have visibility (yes I know some of our older code doesn't have this as well).
2. Read Yii documentation and you will find that the proper way is: Question::model()->findByPk().